src/main/java/dev/jbang/cli/Export.java

```diff
@@ -564,7 +564,8 @@ private void createProjectForExport(BuildContext ctx, Path projectDir) throws IO
 		Util.mkdirs(projectDir);
 
 		// Sources
-		Path srcJavaDir = projectDir.resolve("src/main/java");
+		boolean isKotlin = ctx.getProject().getMainClass().endsWith("Kt");
```

Comment:
this doesn't feel like a great way to detect wether the project is a kotlin project.

`jbang Kt.java` isn't kotlin :)

```suggestion
		boolean isKotlin = cts.getProject().getMainSource() instanceof KotlinSource;
```

Reply:
Fixed